Source/Core/VideoCommon/RenderBase.h

```diff
@@ -184,6 +184,7 @@ class Renderer
   void Swap(u32 xfbAddr, u32 fbWidth, u32 fbStride, u32 fbHeight, const EFBRectangle& rc,
             u64 ticks);
   virtual void SwapImpl(AbstractTexture* texture, const EFBRectangle& rc, u64 ticks) = 0;
+  virtual void OnSwapInvalid() {}
```

Comment:
I'd call this Flush() instead of OnSwapInvalid(). This way it can be used for other purposes too (e.g. flushing the cmdbuffer for deferred EFB copies, once we move that to common).